index.js

```diff
@@ -0,0 +1,3 @@
+const nombreUsuario = prompt("Ingrese su nombre");
+
+alert("Hola "+ nombreUsuario);
```

Comment:
Podés usar interpolación de Strings.
```suggestion
alert(`Hola ${nombreUsuario}`);
```